### ISSUE TITLE
OBS Crashes On Media Deletion

### ISSUE THREAD

**Issue by RodgerOliver** [exeldro/obs-media-controls#10]
## Problem

When I try to delete videos from scenes with the plugin installed, OBS crashes, but i appears to be random.

## Replicating The Problem

- Create 3 scenes
- Add one video to each scene (I used three different videos)
- Delete each one of the videos

### Result

OBS should crash on the last video. Sometimes everything goes fine, sometimes it crashes on the first video. But almost always it crashes.

## Other Problems

One time it crashed when a video was playing and I tried to add a `Video Capture` source to another scene.

Basically OBS was pretty unstable with the plugin.

## Log Files

[OBS Crash 1.txt](https://github.com/exeldro/obs-media-controls/files/5151665/OBS.Crash.1.txt)
[OBS Crash 2.txt](https://github.com/exeldro/obs-media-controls/files/5151661/OBS.Crash.2.txt)
[OBS Crash 3.txt](https://github.com/exeldro/obs-media-controls/files/5151662/OBS.Crash.3.txt)
[OBS Crash 4.txt](https://github.com/exeldro/obs-media-controls/files/5151663/OBS.Crash.4.txt)
[OBS Crash 5.txt](https://github.com/exeldro/obs-media-controls/files/5151664/OBS.Crash.5.txt)

**Comment by maxdrash** [comment]
Can't reproduce issue on ubuntu 20.04.

**Comment by RodgerOliver** [comment]
> Can't reproduce issue on ubuntu 20.04.

The problem was reproduced in two Windows 10 machines.

**Comment by andygadiel** [comment]
Reproduced exact scenario detailed here on macOS Catalina running OSB 25.0.8 (64 bit)

**Comment by exeldro** [comment]
This should be fixed with OBS version 26. Can you test if the release candidate of OBS fixes it?

**Comment by RodgerOliver** [comment]
> 
> 
> This should be fixed with OBS version 26. Can you test if the release candidate of OBS fixes it?

I tested here on OBS Portable 26.0 RC3 and it did not crash.

**Comment by exeldro** [comment]
Now OBS has released version 26 I close this issue.

**Comment by felippenardi** [comment]
It looks like it is still happening on OBS 27 @exeldro
